### DISCUSSION TITLE
Update Release Notes

### DISCUSSION THREAD

**Discussion by MartinX3** [ical4j/ical4j#554] [General]
Please update the release notes
Currently there is v3.2.1 available. But not in the release notes.

Outdated:
- https://github.com/ical4j/ical4j/blob/develop/CHANGELOG.md
- https://ical4j.github.io/docs/ical4j/release-notes

**Comment by benfortuna** [comment]
Release notes on the website are now updated for 3.2.0 and 3.2.1 releases.

https://ical4j.github.io/docs/ical4j/release-notes
